recipes/libsodium/1.0.18/conanfile.py

```diff
@@ -0,0 +1,223 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools, MSBuild
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class LibsodiumConan(ConanFile):
+    name        = "libsodium"
+    version     = "1.0.18"
+    description = "A modern and easy-to-use crypto library."
+    license     = "ISC"
+    url         = "https://github.com/conan-io/conan-center-index"
+    homepage    = "https://download.libsodium.org/doc/"
+    exports_sources = ["patches/**"]
+    settings    = "os", "compiler", "arch", "build_type"
+    topics = ("sodium", "libsodium", "encryption", "signature", "hashing")
+    generators  = "cmake"
+    _source_subfolder = "source_subfolder"
+
+    options = {
+        "shared" : [True, False],
+        "fPIC": [True, False],
+        "use_soname" : [True, False],
+        "use_pie"    : [True, False],
```

Comment:
Should the name of the option be just `PIE` to follow the same convention as `fPIC`?